meson.build

```diff
@@ -24,6 +24,7 @@ py_src = {
   'euphonic/data': ['__init__.py', 'bluebook.json', 'sears-1992.json',
                     'reciprocal_spectroscopy_definitions.txt'],
   'euphonic/readers': ['__init__.py', 'castep.py', 'phonopy.py'],
+  'euphonic/spectra': ['__init__.py', 'base.py', 'collections.py'],
```

Comment:
Nice, glad this is moderately manageable even if you do have to manually track the modules.